context/config.ts

```diff
@@ -1,12 +1,12 @@
 import { NetworkType } from "@airgap/beacon-sdk";
 
 export const RPC_URL =
-  process.env.NEXT_PUBLIC_RPC_URL ?? "https://ghostnet.tezos.marigold.dev/";
+  process.env.NEXT_PUBLIC_RPC_URL ?? "https://ghostnet.ecadinfra.com/";
```

Comment:
I think we should change back to the marigold's rpc

Reply:
I updated it when marigold was down, and I forgot to switch it back